synapse/res/templates/sso_redirect_confirm.html

```diff
@@ -2,35 +2,50 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
-    <title>SSO redirect confirmation</title>
+    <title>Your account is now ready</title>
```

Comment:
@bwindels this will be shown for existing users as well as new users: is it correct?

Reply:
Indeed, we probably want to change this to "Continue to your account" for both cases.

Reply:
done in 79de3283bc